File: SampleManageSystem/Model/Privilege.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Model
{
    public class Privilege:BaseModel
    {
        public Privilege()
        {

        }
        public Privilege(int privilegeid,int moduleid,string name,bool canread,bool cancreate,bool candelete,bool canwrite,bool canmanage ,string modulename,string tablename)
        {
            this.PrivilegeId = privilegeid;
            this.Name = name;
            this.ModuleName = modulename;
            this.CanRead = canread;
            this.CanCreate = cancreate;
            this.CanDelete = candelete;
            this.CanWrite = canwrite;
            this.ModuleId = moduleid;
            this.CanManage = canmanage;
            this.Tablename = tablename;
        }
      
        public int PrivilegeId { get; set; }
        public string Name { get; set; }
        public string ModuleName { get; set; }
        public bool CanRead { get; set; }
        public bool CanCreate { get; set; }
        public bool CanDelete { get; set; }
        public bool CanWrite { get; set; }
        public int ModuleId { get; set; }
        public bool CanManage { get; set; }
        public string Tablename { get; set; }
    }
}

File: ManageSystem/js/control/privilege.js
function privilege() {
    (function () {
        $.ajax({
            type: "post",
            contentType: "application/json;charset=utf-8",
            url: Globals.ServiceUrl + "GetPrivilegeList",
            success: function (data) {
                var s = JSON.parse(data.d);
             
                    var tbody = $(".table tbody").empty();
                    for (var i in s) {
                        var cont = "<td>" + s[i].Name + "</td><td>" + s[i].ModuleName+ "</td><td>" +istrue(s[i].CanRead)+ "</td><td>"
                                   + istrue(s[i].CanDelete) + "</td><td>" + istrue(s[i].CanWrite) + "</td><td>" + istrue(s[i].CanCreate) + "</td><td>" + istrue(s[i].CanManage) +
                                   "<ul class='actions'><li class='last'><a class='edit'>编辑</a> <a class='delete'>删除</a></li></ul>" + "</td><td name='privilegeid' style='display:none'>"+s[i].PrivilegeId+"</td>"
                        var row = document.createElement("tr");
                        row.innerHTML = cont;
                        tbody.append(row);                  
                    }
                    $(".edit").click(function () {
                        location.href = "privilege-profile.html?id=" + $(this).parent().parent().parent().parent().find("[name=privilegeid]").text();
                    })
                    $(".delete").click(function () {
                        if (confirm("确定删除？")) {
                            var jsonPar = {
                                privilegeid: $(this).parent().parent().parent().parent().find("[name=privilegeid]").text()
                            }
                            $.ajax({
                                type: "post",
                                contentType: "application/json; charset=utf-8",
                                url: Globals.ServiceUrl + "DeletePrivilege",
                                data: JSON.stringify(jsonPar),
                                success: function (data) {
                                    var s = JSON.parse(data.d);
                                    if (s) {
                                        alert("删除成功");
                                        window.location.reload();
                                    } else {
                                        alert("删除失败");
                                    }
                                }
                            })
                        }
                      
                    })
             
            }, error: function (xhr) {
                 alert("请联系管理员");
            }
        })
      
    })()
}

function istrue(bool) {
    if (bool) {
        return '✔';
    } else {
        return '✘';
    }
}
$(function () {
    privilege();
})